### PR TITLE
New version: JetBrains.Hub version 22.1.14638

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58578)

### CODE REVIEW

File: manifests/j/JetBrains/Hub/22.1.14638/JetBrains.Hub.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+PackageVersion: 22.1.14638
+PackageLocale: en-US
+Publisher: JetBrains
+PublisherUrl: https://www.jetbrains.com
+PublisherSupportUrl: https://www.jetbrains.com/help/hub/Introduction-to-HUB.html
+PrivacyUrl: https://www.jetbrains.com/privacy-security
+Author: JetBrains s.r.o.
+PackageName: JetBrains Hub
+PackageUrl: https://www.jetbrains.com/hub
+License: https://www.jetbrains.com/hub/
```
Comment (Trenly) [inline manifests/j/JetBrains/Hub/22.1.14638/JetBrains.Hub.locale.en-US.yaml:14]: License doesn't seem quite right

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/j/JetBrains/Hub/22.1.14638/JetBrains.Hub.locale.en-US.yaml (hunk 1 above)
Comment (vedantmgoyal2009) [reply manifests/j/JetBrains/Hub/22.1.14638/JetBrains.Hub.locale.en-US.yaml:14]: What should I keep? Proprietary?

https://www.jetbrains.com/help/hub/third-party-software.html#Hub_Front-End_Libraries

Comment (jedieaston) [inline manifests/j/JetBrains/Hub/22.1.14638/JetBrains.Hub.locale.en-US.yaml:14]:
```suggestion
License: Proprietary
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
That shouldn't be a link...

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**